Update The Attack Surface documentation (#3936)
MicrosoftDocs/windows-itpro-docs: windows/security/threat-protection/windows-defender-exploit-guard/attack-surface-reduction-exploit-guard.md

1121 | Event when rule fires in Block-mode
1122 | Event when rule fires in Audit-mode

> The engine version of the attack surface reduction events in the event log, is part of the Widnows Defender product, not of the Oeprating System, but because Windows Defender is integrated with Windows 10 Operating System, this feature works on all machines with Windows 10 installled.

Reviewer: ```suggestion
The "engine version" of attack surface reduction events in the event log, is part of  Windows Defender ATP, not of the operating system. Windows Defender ATP is integrated with Windows 10, so this feature works on all machines with Windows 10 installed.
```
